release notes/v0.48.0.md

```diff
@@ -0,0 +1,256 @@
+k6 v0.48.0 is here 🎉! This release includes:
+
+- Numerous long-awaited breaking changes.
+- A new `k6 new` subcommand to generate a new test script.
+- A new `k6/experimental/fs` module for file interactions.
+- k6 browser got a support for CPU and network throttling.
```

Comment:
```suggestion
- The k6 browser module received support for CPU and network throttling.
```

Reply:
Now, I feel that it's better to rewrite this sentence, to be aligned with the `This release includes:`
```suggestion
- A CPU and network throttling support for the k6 browser.
```

How does this new version sound to you?

Reply:
How about?:

```suggestion
- CPU and network throttling support for the k6 browser.
```